Fluentd for Audit Logging (#2853)
antrea-io/antrea: docs/antrea-network-policy.md

    test-anp   securityops   5          5s
```

> ### Audit logging for Antrea-native policies

Reviewer: i think we can place this right after ClusterGroup section.. this seems like an odd place

Logs are recorded in `/var/log/antrea/networkpolicy` when `enableLogging` is configured.

> Fluentd could be used to assist with analyzing the logs. To see a documentation, refer to `antrea/docs/cookbooks/fluentd`.

Reviewer: ```suggestion
Fluentd could be used to assist with analyzing the logs. To see a documentation, refer to [Fluentd cookbook](cookbooks/fluentd).
```

    test-anp   securityops   5          5s
```

> ### audit-logging-for-antrea-networkpolicy

Reviewer: ```suggestion
### Audit Logging for Antrea-native policies
```

> The following commands can be used to audit Antrea networkpolicy using Fluentd

Reviewer: ```suggestion
The following commands can be used to audit Antrea-native policies using Fluentd
```

to collect logs and send to Elasticsearch to display on Kibana dashboard.

```bash
    # Deploy Elasticsearch and Kibana dashboard
    kubectl apply -f build/yamls/audit-logging/kibana-elasticsearch.yml
    

>     # Set rbac for Fluentd

Reviewer: ```suggestion
    # Set RBAC for Fluentd
```

    kubectl get cg.crd.antrea.io
```

## Audit logging for Antrea-native policies

Logs are recorded in `/var/log/antrea/networkpolicy` when `enableLogging` is configured.

> Fluentd could be used to assist with analyzing the logs. To see a documentation, refer to [Fluentd cookbook](cookbooks/fluentd).

Reviewer 2: ```suggestion
Fluentd can be used to assist with analyzing the logs. Refer to the [Fluentd cookbook](cookbooks/fluentd) for documentation.
```
